docs/hyperchains.md

```diff
@@ -340,6 +348,347 @@ You can now begin testing transactions and interactions with your Hyperchain dep
+
+```shell
+docker run --rm -it -p 33013:3013 \
+    --network hyperchain \
+    -v ${PWD}/hc_test/nodeConfig/validator2.yaml:/home/aeternity/.aeternity/aeternity/aeternity.yaml \
+    -v ${PWD}/hc_test/nodeConfig/hc_test_accounts.json:/home/aeternity/node/data/aecore/hc_test_accounts.json \
+    -v ${PWD}/hc_test/nodeConfig/hc_test_contracts.json:/home/aeternity/node/data/aecore/hc_test_contracts.json \
+    aeternity/aeternity:v7.3.0-rc3
+```
+
+Verify the new validator node is running with:
+```shell
+curl -s localhost:33013/v3/status | jq
+```
+
+Expected output:
+```yaml
+{
+  "difficulty": 0,
+  "genesis_key_block_hash": "kh_7dm2zSo6NsnEDMYBYdXA9QvkJvvk7TenT68HxW5BSrRSz3WV6",
+  "hashrate": 0,
+  "listening": true,
+  "network_id": "hc_test",
+  "node_revision": "57bc00b760dbb3ccd10be51f447e33cb3a2f56e3",
+  "node_version": "7.3.0-rc3",
+  "peer_connections": {
+    "inbound": 0,
+    "outbound": 1
+  },
+  "peer_count": 1,
+  "peer_pubkey": "pp_ENJfMAPXWbZruYgexnGCNEwCBwdLcEiR8F541CuzCkgtFQ5jt",
+  "pending_transactions_count": 0,
+  "protocols": [
+    {
+      "effective_at_height": 0,
+      "version": 6
+    }
+  ],
+  "solutions": 0,
+  "sync_progress": 100,
+  "syncing": false,
+  "top_block_height": 35,
+  "top_key_block_hash": "kh_FK7EvnCYsq9DdbZLtBpFPdD5UdTNYGWnWfQShNgVkbY6iyP3h",
+  "uptime": "59s.448"
+}
+```
+
+Make sure that:
+
+- `genesis_key_block_hash` is the same as the status output of the initiator node
+- `network_id` is the same as the status output of the initiator node
+- `peer_count` is more than 0
+- `top_block_height` is more than 0
+- `sync_progress` is 100
+
+The above should confirm the new validator is up, running and connected to the initiator Hyperchain network!
+
+### Register New Validator
+
+Once the new node is running it's time to register it as validator with the Hyperchain consensus.
+
+#### CLI Node Config
+First the CLI have to be configured to work with the locally running Hyperchain network created in this example:
+```shell
+aecli select-node http://localhost:3013/
+export AECLI_NODE_URL=http://localhost:3013/
+```
+
+Verify the configuration by running:
+```shell
+aecli config
+```
+
+Expected output:
+```shell
+Node http://localhost:3013/ network id hc_test, version 7.3.0-rc3, protocol 6 (Ceres)
+Compiler https://v8.compiler.aepps.com/ version 8.0.0
+```
+
+#### Funding
+
+**The staker account must be funded with at least the minimum staking amount (1000000AE in this example) and some extra for transacting. In this example the treasury account can be used by loading it in a wallet**
+
+With the help of the CLI the treasury secret key is imported then used to send tokens to the new staker account:
+
+```shell
+aecli account create hc_test/wallets/treasury.js 9ab3bb473a0ec30d4f75f877ee9de0145fbd883e07edab2e4555e156a729be5619bd0a236a9ebd4a47e5fc3961b59ea215515109c007e2ce844b1f543d4e9598
```

Comment:
```suggestion
aecli account create hc_test/wallets/treasury.js sk_2B8f6qsTGj9B3SG1f4D8u1zhDhaHtUovRERiPECyzbHDBbriHW
```
🤷‍♀️

Reply:
I'll keep it as it for now, seems to work.